scholia/app/views.py

```diff
@@ -130,6 +131,117 @@ def redirect_q(q):
     return redirect(url_for(method, q=q), code=302)
 
 
+@main.route("/" + q_pattern + "/bioschemas")
+def bioschemas_for(q):
+    """Detect a Bioschemas API request and return Bioschemas for this item.
+
+    Parameters
+    ----------
+    q : str
+        Wikidata item identifier
+    """
+    entity_class = q_to_class(q)
+
+    # get the aspect specific bits
+    if entity_class == "author":
+        data = {
+            "@type": "Person",
+            "http://purl.org/dc/terms/conformsTo": {
+                "@type": "CreativeWork",
+                "@id": "https://bioschemas.org/profiles/Person/0.2-DRAFT-2019_07_19/",
+            },
+            "description": "A person",
```

Comment:
@egonw consider using the wikidata description for the person?

Reply:
yes, I have :) The description goes for most of the items, but I really first need to know if @fnielsen likes the idea of proxying before I start tweaking stuff

Reply:
descriptions are now added.